### PR TITLE
add homotopy group

### PR DESCRIPTION
**Pull request by riaqn**

### CODE REVIEW

**Comment by Alizter** [comment]
Why only left axioms?

**Comment by riaqn** [comment]
@Alizter Actually the part about groups is in the other PR #103 

but basically, the right ones can be derived from the left ones: 
https://github.com/HoTT/HoTT-Agda/blob/1037d82edcf29b620677a311dcfd4fc2ade2faa6/core/lib/types/Group.agda#L33

File: Cubical/Data/HomotopyGroup/Base.agda [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+module Cubical.Data.HomotopyGroup.Base where
+
+open import Cubical.Core.Prelude
+open import Cubical.Data.Nat
+open import Cubical.HITs.SetTruncation
+open import Cubical.Foundations.HLevels
+open import Cubical.Data.Group.Base
+import Cubical.Foundations.GroupoidLaws as GL
+
+Pointed : ∀ {ℓ} → Set (ℓ-suc ℓ)
```
Comment (mortberg) [inline Cubical/Data/HomotopyGroup/Base.agda]: We might want to make a separate file for pointed types in Foundations?

Comment (Alizter) [reply Cubical/Data/HomotopyGroup/Base.agda]: @mortberg Didn't #105 add some pointed stuff too?

Comment (mortberg) [reply Cubical/Data/HomotopyGroup/Base.agda]: Yes, some of that should be upstreamed to the proper file.

**Comment by riaqn** [comment]
@mortberg  I just realized how cumbersome it is to develop this in separate branches - every time I need something in group for EM space, I need to make that commit in the group branch, and let hgroup merge the group branch, and let emspace branch merge the hgroup branch. It's just too much work.

May I just have one branch that contains all works regarding group, homotopy group, and emspace? It would be much easier for me.

**Comment by mortberg** [comment]
> @mortberg I just realized how cumbersome it is to develop this in separate branches - every time I need something in group for EM space, I need to make that commit in the group branch, and let hgroup merge the group branch, and let emspace branch merge the hgroup branch. It's just too much work.
> 
> May I just have one branch that contains all works regarding group, homotopy group, and emspace? It would be much easier for me. It would be harder for you to review, tho.

If you fix the few comments on https://github.com/agda/cubical/pull/103 I can merge it and there will only be one PR. If you then also fix the comments on this PR I can merge it as well. Your branch about EM spaces will then be based on master and you won't have to merge anything except rebasing on master now and then.

**Comment by Alizter** [comment]
You can merge those branches into your branch, when they are merged seperately into main you will have to rebase your branch but it shouldn't be too difficult to do.

**Comment by Alizter** [comment]
Why isn't travis happy?

**Pull request merged**